fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataService.java

```diff
@@ -73,4 +72,7 @@ public interface ReadWriteNonCoreDataService {
     String getDataTableName(String Url);
 
     Long countDatatableEntries(String datatableName, Long appTableId, String foreignKeyColumn);
+
+    List<JsonObject> queryDataTable(@NotNull String datatable, @NotNull String columnName, String columnValue,
```

Comment:
is there a reason to change the order of the method and change the name of the parameters?